OpenRA.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 2013
+VisualStudioVersion = 12.0.31101.0
+MinimumVisualStudioVersion = 10.0.40219.1

```

Comment:
This sets the minimum supported version to Visual Studio 2010, so I think we can live with that. I should check how the line above this affects my Visual Studio 2012 though, because you are setting the current version to 2013.